### PR TITLE
Add monitor keyboard shortcut text ~ 

### PR DESCRIPTION
**Pull request by MasonT8198**
Rather minor PR, however this adds the ~ text next to the monitor button as posted in the #showcase channel of xemu,

### CODE REVIEW

**Comment by MasonT8198** [comment]
image

File: ui/xemu-hud.cc [hunk 1]
```diff
@@ -2066,6 +2066,8 @@ static void process_keyboard_shortcuts(void)
 #define SHORTCUT_MENU_TEXT(c) "Ctrl+" #c
 #endif
 
+#define SHORTCUT_MENU_KEY(c) "" #c
```
Comment (mborgerson) [inline ui/xemu-hud.cc]: Why the `""` empty string?

**Comment by MasonT8198** [comment]
@mborgerson fixed

**Comment by mborgerson** [comment]
Thanks!

**Pull request merged**